testnet/stacks-node/src/nakamoto_node/miner.rs

```diff
@@ -1082,13 +1077,11 @@ impl BlockMinerThread {
             ) {
                 error!("Relayer: Failure mining anchored block: {e}");
             }
-            NakamotoNodeError::MiningFailure(e)
+            e
```

Comment:
Unrelated to this PR, but the 3 lines above - why not just always have that `error!` log?

Reply:
I think that's because those two types of errors are normal and happen often, so we don't want to report an error in those cases.